From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 15 Jul 2018 21:53:39 +0100
Subject: Never clean files in quilt status directory
Forwarded: https://www.zytor.com/pipermail/klibc/2018-July/003999.html

The top-level "distclean" and "mrproper" targets delete zero-size
files in the source, perhaps assumed to be stamp files.  This is fine
for actual source direcgtories, but disastrous for a quilt status
directory.

For each patch, quilt stores the previous versions of the modified
files in a subdirectory of .pc.  If the patch creates a file, this
file will be empty.  Removing it effectively removes the file creation
from the patch.

The .git directory is already excluded from the clean rules for
a similar reason, so do the same for .pc.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/Makefile
+++ b/Makefile
@@ -158,7 +158,7 @@ FORCE: ;
 ###
 # clean: remove generated files
 # mrproper does a full cleaning including .config and linux symlink
-FIND_IGNORE := \( -name .git \) -prune -o
+FIND_IGNORE := \( -name .git -o -name .pc \) -prune -o
 quiet_cmd_rmfiles = $(if $(wildcard $(rm-files)),RM     $(wildcard $(rm-files)))
       cmd_rmfiles = rm -f $(rm-files)
 clean:
